src/mumble/JackAudio.h

```diff
@@ -28,7 +28,9 @@ struct jack_ringbuffer_data_t {
 };
 
 typedef QVector< jack_port_t * > JackPorts;
+typedef QHash< QString, jack_port_t * > JackNamedPorts;
 typedef QVector< jack_default_audio_sample_t * > JackBuffers;
+typedef QVector< jack_ringbuffer_t * > JackRingBuffers;
```

Comment:
Also here: Please prefer STL types in new code, unless you have a good reason not to (e.g. because you then would have to convert it to Qt types anyway

Reply:
JackRingBuffers doesn't end up getting used, and I can certainly use the STL equivalent for JackNamedPorts. Will fix